l10n_pt_stock_vehicle_daily/models/stock_picking.py

```diff
@@ -1,10 +1,21 @@
 # Copyright 2024 Open Source Integrators
+# Copyright 2025 NuoBiT Solutions - Deniz Gallo <dgallo@nuobit.com>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class StockPicking(models.Model):
     _inherit = "stock.picking"
 
-    l10n_pt_license_plate = fields.Char("License Plate")
+    l10n_pt_license_plate = fields.Char(
+        string="License Plate",
+        compute="_compute_l10n_pt_license_plate",
+        store=True,
+        readonly=False,
+    )
+
+    @api.depends("location_id")
+    def _compute_l10n_pt_license_plate(self):
+        for picking in self:
+            picking.l10n_pt_license_plate = picking.location_id.l10n_pt_license_plate
```

Comment:
Can this be a simple related field?

Reply:
He did that way because is how it's done in the Wizard of the same module

https://github.com/OCA/l10n-portugal/blob/4624765d532c38dc815adad1a4577935a4638528/l10n_pt_stock_vehicle_daily/wizard/stock_move_location.py#L33-L38